packages/importer/test/columns.test.ts

```diff
@@ -294,6 +305,96 @@ const coerceTestCases: { input: unknown; type: Schemas.Column['type']; options?:
       type: 'datetime',
       expected: { value: new Date('2020-01-01T00:00:00+01:00'), isError: false }
     },
+    // This test should have the response mocked
```

Comment:
This should mock the fetch response so that we don't need a stable remote URL to check against